frappe/public/js/frappe/ui/toolbar/toolbar.js

```diff
@@ -200,6 +200,7 @@ $.extend(frappe.ui.toolbar, {
 
 frappe.ui.toolbar.clear_cache = frappe.utils.throttle(function() {
 	frappe.assets.clear_local_storage();
+	frappe.assets.clear_indexed_db();
```

Comment:
Clearing IndexedDB on a hard refresh will clear all saved drafts, I don't think this is the expected behaviour. `clear_local_storage` also only clears certain keys, saved drafts were not cleared earlier.

Reply:
On the other hand, there's no way to "discard" saved drafts.

Reply:
Would the button be used to discard all saved drafts or just the current one? If it's the former, it wouldn't be very intuitive to have that button on the email popup (since the context is just the current email). Discarding just the current email wouldn't make much sense either.

Maybe we can add an action to discard all drafts to the menu in the Email Inbox view instead? (It wouldn't be very visible though)

Reply:
I'm curious.. Why would discarding the current draft not make sense?

It's what Gmail does:

![Screenshot-2021-04-08-114129](https://user-images.githubusercontent.com/16315650/113977035-9855b180-985f-11eb-85fd-792aecc0c0ba.png)